.circleci/config.yml

```diff
@@ -1,63 +1,91 @@
-version: 2
+version: 2.1
+
+orbs:
+  python: circleci/python@1.2.0
 
-workflows:
-  version: 2
-  test:
-    jobs:
-      - build-python-3.6
-      - build-python-3.7
-      - build-python-3.8
-      - build-python-3.9
 
 jobs:
-  build-python-3.6: &template
+  pre-build:
+    description: Install and run a Python standalone package
+    parameters:
+      command-name:
+        type: string
+      command-run:
+        type: string
     docker:
-      - image: python:3.6
+      - image: cimg/python:3.7
```

Comment:
If we're running flake8/black/mypy in just one Python version, I am wondering if we can use `cimg/python:3.9` here since Python 3.9 has just come out and using it here would make this config as future proof as possible. Is Python 3.7 chosen here for specific reasons?

Reply:
No, I had no specific reason in mind; I just chose a chose a version I knew it worked.